# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/shell/jsshell.h
# Commit: a33765bea383
# Full Hash: a33765bea383ee30312063a08733324b65dc58be
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2018-05-25 17:52:14
# Description:
#   Bug 1463371 - Revert to previous behaviour for setting module resolve hook in the shell r=luke
# ==============================================================================

diff -r d99b7e4e8cd9 -r a33765bea383 js/src/shell/jsshell.h
--- a/js/src/shell/jsshell.h	Fri May 25 11:12:04 2018 +0200
+++ b/js/src/shell/jsshell.h	Fri May 25 10:39:40 2018 +0100
@@ -180,8 +180,6 @@
     // Off-thread parse state.
     js::Monitor offThreadMonitor;
     Vector<OffThreadJob*, 0, SystemAllocPolicy> offThreadJobs;
-
-    JS::PersistentRootedFunction moduleResolveHook;
 };
 
 extern ShellContext*
